packages/thirdweb/src/react/web/ui/MediaRenderer/MediaRenderer.tsx

```diff
@@ -107,28 +108,8 @@ export const MediaRenderer = /* @__PURE__ */ (() =>
 
         // 3d model
         if (mediaInfo.mimeType.startsWith("model")) {
-          return (
-            <Suspense
-              fallback={
-                poster ? (
-                  <img
-                    style={mergedStyle}
-                    src={poster}
-                    alt={alt}
-                    ref={ref as unknown as React.LegacyRef<HTMLImageElement>}
-                    className={className}
-                  />
-                ) : null
-              }
-            >
-              <ModelViewer
-                style={mergedStyle}
-                src={mediaInfo.url || ""}
-                poster={poster}
-                alt={alt}
-                className={className}
-              />
-            </Suspense>
+          console.error(
+            "Encountered an unsupported media type. 3D model support was removed in v5.92.0.",
```

Comment:
I would instead give the dev a hint / link on what to do. Link to Model viewer repo and say something like: "to render 3d models, please use ModelViewer from this repo"